### DISCUSSION TITLE
upload组件支持通过js手动上传多个文件吗

### DISCUSSION THREAD

**Discussion by qlly0630** [ant-design/ant-design#42617] [Q&A]
场景是用户上传一张长图的时候，需要拦截这张长图，然后将长图裁剪成多张图片，再上传

**Comment by ChenKun1997** [comment]
上传不就是调接口吗，在下面这两属性里面应该都可以吧
![image](https://github.com/ant-design/ant-design/assets/24249578/ac6ac520-0402-4432-8555-ce3bf75c6504)

**Reply by qlly0630** [reply]
beforeUpload只能上传一个文件吧，在beforeUpload拦截后，需要上传多个裁剪后的文件

**Reply by ChenKun1997** [reply]
> beforeUpload只能上传一个文件吧，在beforeUpload拦截后，需要上传多个裁剪后的文件

函数里面裁剪，promise.all同时传裁剪的不就行了，自己调用上传图片接口，不要让组件替你调

**Reply by qlly0630** [reply]
> > beforeUpload只能上传一个文件吧，在beforeUpload拦截后，需要上传多个裁剪后的文件
> 
> 函数里面裁剪，promise.all同时传裁剪的不就行了，自己调用上传图片接口，不要让组件替你调

我试过在beforeUpload存储，在customRequest内上传，可以实现上传的功能，但怎么做展示呢
